src/lib/network/swagger-client.ts

```diff
@@ -757,7 +757,9 @@ export type RequestParams = Omit<FullRequestParams, "body" | "method" | "query"
 export interface ApiConfig<SecurityDataType = unknown> {
   baseUrl?: string;
   baseApiParams?: Omit<RequestParams, "baseUrl" | "cancelToken" | "signal">;
-  securityWorker?: (securityData: SecurityDataType | null) => Promise<RequestParams | void> | RequestParams | void;
+  securityWorker?: (
+    securityData: SecurityDataType | null
+  ) => Promise<RequestParams | void> | RequestParams | void;
```

Comment:
This file is auto generated, we shouldn't edit it manually. It looks like you ran a formatter on this, that is not necessary.. Please check, maybe we should ignore formatting here

Reply:
Yes. Reverted it. Regenerated and didn't run formatter on it.